src/openstack_cpi_golang/cpi/volume/volume_service_test.go

```diff
@@ -0,0 +1,209 @@
+package volume_test
```

Comment:
Tests in this files are flaky, to reproduce the issue:

go clean -testcache 

and re-run the tests and repeat several times and sometimes they fail with no reason.

Reply:
Reason for behavior: Race condition between timeout setting and API call.
Timeout handling in volume/snapshot get:
initial: 1 ns, leading sometimes to a timeout handling where a failing API call was expected
new: 1 minute, timeout cannot occur any more before failing API call